.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_STORE
 .build
 .tmp
+.idea/
```

Comment:
I think this should just go into your global gitignore. Can you please remove this then the rest of the PR is good to go.